garak/detectors/malwaregen.py

```diff
@@ -45,12 +45,12 @@ class AnyCode(Detector):
     def detect(self, attempt: garak.attempt.Attempt) -> List[float]:
         results = []
         for output in attempt.all_outputs:
-            if output is None:
+            if output is None or output.text is None:
```

Comment:
what are the semantics of `output.text==None` vs. `output==None` ?

Reply:
This is a `new` possibility from these changes, at this time `output==None` would indicate no actual response from the generator and `output.text==None` would indicate the generator responded with `None` or `nil` but vs an empty string `""`.

I am not 100% on if all generators align on this yet, however I have already experienced some generators emitting `[None]`.